Environment/TestLocalDockerEnvironment.py

```diff
@@ -15,7 +15,7 @@
   - Error handling here is minimal; in production wrap execute calls and inspect return values.
```

Comment:
Please move all the test to tests to the tests directory. Let's maintain same directory structure inside the `tests` directory also. The `<class>.test.py` file should be named after the python file it is testing. And the directory should be same as the original class file from the project root.

Rationale: I'm still not sure where to keep the tests (next to the code itself like this or keep all tests separately - in the test folder). I'm just deciding for time-being to keep the tests away from cognitive overload. 😕